scm-ui/ui-webapp/src/repos/sources/containers/Content.tsx

```diff
@@ -129,6 +156,7 @@ class Content extends React.Component<Props, State> {
           </div>
           <div className="buttons is-grouped">
             {selector}
+            <OpenInFullscreenButton onClick={this.openModal} />
```

Comment:
Why not use the extension point which is right beneath? I know you will need to expose more props to the extension point but it should be okay. Maybe I am missing something. Let's also talk about this.